src/Microsoft.TestPlatform.CrossPlatEngine/Client/ProxyOperationManager.cs

```diff
@@ -152,6 +151,7 @@ public virtual void Initialize(bool skipDefaultAdapters)
             string runSettings,
             ITestMessageEventHandler eventHandler)
         {
+            // NOTE: Event handler is ignored here, but it is used in the overloaded method.
             return this.SetupChannel(sources, runSettings);
```

Comment:
Looked like this to me too before I investigated and left this comment for future reference, even though the initial changes were also done by me.

The events handler is being used by the `SetupChannel` method in `ProxyOperationManagerWithDataCollection`. The reasoning behind having this events handler here even though it's not used is to have a consistent interface for when having to call `SetupChannel` because we don't know if `GetTestSessionManager` will return a `ProxyOperationManager` or a `ProxyOperationManagerWithDataCollection`.